### ISSUE TITLE
High heterozygous?

### ISSUE THREAD

**Issue by shehongbing** [schatzlab/genomescope#118]
Hi,
     i estimate the genome using HiFi reads and the obtain high heterozygous (7%). However, i can not infer the two obviously peaks.

![linear_plot](https://github.com/schatzlab/genomescope/assets/49432965/58124d9e-20aa-4509-9deb-3bf1dd13ede7)
![linear_plot](https://github.com/schatzlab/genomescope/assets/49432965/b1d3e258-dae1-4a77-be87-1d7f7057ca98)

**Comment by mschatz** [comment]
I agree the fit is questionable. Do you have any information about the
species or the assembly? Can you send the link to the genomescope webpage
(or send the histo file)

Thanks!
Mike

**Comment by mschatz** [comment]
Hi Hongbing

I agree there was a bad fit with the model. I can get a slightly better fit
by setting "Average k-mer coverage for polyploid genome" to 9. This gives
the model fitting algorithm some help in determining which peak is which.
With this parameter, the heterozygosity drops to almost nothing and the
genome size increases to 1.4Gb. But this fit still isnt as good as I
would like and there is still a fairly large gap between the model and the
observed counts. I think it would really help to use deeper coverage. Is
this at all possible?

Here are the updated results:
http://qb.cshl.edu/genomescope/genomescope2.0/analysis.php?code=GL5JdZG5ro7wKPbGfxGj

Good luck!
Mike
